docs/references/react/use-organization.mdx

```diff
@@ -28,13 +28,29 @@ const {
 To keep network usage to a minimum but to also provide a smooth auto-updating experience to developers, we made the `invitationList` attribute available only when its respective parameter is present when calling the hook.
```

Comment:
Do we still need this if `invitationList` is now deprecated?

Reply:
it is, but this needs to cover all, `invitations`, `domains` .... so i'll give it another look.